_data/versions.yml

```diff
@@ -1,6 +1,9 @@
 # vX.Y format: update during major release process
 currentReleaseStream: v2.5
 
+# Local directories to ignore when checking external links
+htmlProoferLocalDirIgnore: /v1.5/,/v1.6/,/v2.0/,/v2.1/,/v2.2/
```

Comment:
It'd be cool to see this get passed as a boolean field on each version below. But I understand this is a reasonable increment for now so lgtm

Reply:
this is a good idea for when/if we fully automate maintenance of versions.yml file (via Ruby?)